src/mq.js

```diff
@@ -57,8 +57,8 @@ define(['ModernizrProto', 'injectElementWithStyles'], function(ModernizrProto, i
       var bool = false;
 
       injectElementWithStyles('@media ' + mq + ' { #modernizr { position: absolute; } }', function(node) {
-        bool = (window.getComputedStyle ?
-          window.getComputedStyle(node, null) :
+        bool = (computedStyle ?
```

Comment:
in what circumstance would `computedStyle ` be falsey? its an imported function

Reply:
you're right, totally screwed up the usage here :-)